### PR TITLE
Allow the Apache HTTP instrumentation image to be used for non-root users

### LINKED ISSUE #2068: Apache HTTPD autoinstrumentation cannot be run with non-root user because of the permissions in the instrumentation image
For the Apache HTTPD autoinstrumentation, we copy some stuff from one of the init containers. When using an user different to root, the files cannot be copied (because the user will not have permissions to that).

### PR DESCRIPTION
**Pull request by iblancasa**
Resolves #2068.

### CODE REVIEW

File: autoinstrumentation/apache-httpd/Dockerfile [hunk 1]
```diff
@@ -20,6 +20,7 @@ FROM alpine:latest
 
 COPY --from=agent /opt/opentelemetry/agent/opentelemetry-webserver-sdk /opt/opentelemetry
 
+RUN chmod 775 -R /opt/opentelemetry/
```
Comment (TylerHelmuth) [inline autoinstrumentation/apache-httpd/Dockerfile:23]: @chrlic can you confirm this change is safe.

Comment (iblancasa) [reply autoinstrumentation/apache-httpd/Dockerfile:23]: If this fix is not fine, I'll be happy implementing another approach but... I'll need some hints 😅

Comment (TylerHelmuth) [reply autoinstrumentation/apache-httpd/Dockerfile:23]: Reading through the issue, I think you're right that this is totally fine.

**Review by TylerHelmuth — APPROVED** [review]

File: autoinstrumentation/apache-httpd/Dockerfile (hunk 1 above)
Comment (chrlic) [reply autoinstrumentation/apache-httpd/Dockerfile:23]: Yes this is safe. I'd do it the same way.

**Review by pavolloffay — APPROVED** [review]

**Pull request merged**